registry/flutter_cocoon.test

```diff
@@ -0,0 +1,7 @@
+contact=flutter-infra@google.com
+fetch=git clone https://github.com/flutter/cocoon.git tests
+fetch=git -C tests checkout b5f4a23ed535ce4fed77e3c1124643e0838f6fdc
+update=dashboard
+# Runs flutter analyze, flutter test, and builds web platform
+test.posix=./test_utilities/bin/flutter_test_runner.sh dashboard
```

Comment:
Is it intentional to test dashboard only?

Reply:
Yes, as it's the minimum needed to get the test coverage in Flutter CI. If we're interested, I can send some Cocoon PRs to make app_dart compatible with the customer test suite.

Reply:
It would be helpful to catch some potential regressions so that cocoon would not be broken.

But I guess last time why we removed the cocoon test from here is: the intended changes caused cocoon regression. From flutter CI: it cannot pass due to downstream cocoon hasn't picked up this change;
From cocoon side, it cannot pick up the change because the flutter PR hasn't landed yet.

How do we workaround that issue when it happens again?

Reply:
That would be a breaking change. Cocoon uses tip of tree Flutter, so any change like that would need to be made a soft-change.

In the past, contributors would temporarily disable the tests for a day to land their change, but that was moreso for dealing with lint changes.

Reply:
Yeah, let's enable only dashboard for now.